pkgs/development/web/nhost/default.nix

```diff
@@ -0,0 +1,29 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "nhost";
+  version = "1.5.2";
+
+  src = fetchFromGitHub {
+    owner = "nhost";
+    repo = "cli";
+    rev = "v${version}";
+    sha256 = "sha256-mjhNZwbYc5RKxDBMmXCllKhHNJYwau5rQABUjKYIkf0=";
+  };
+
+  vendorSha256 = null;
+
+  postInstall = ''
+    mv $out/bin/cli $out/bin/nhost
+  '';
+
+  meta = with lib; {
+    description = "The Nhost CLI is used to get a local development environment.";
+    homepage = "https://github.com/nhost/cli";
+    license = licenses.mit;
+    maintainers = with maintainers; [ andr-ec ];
```

Comment:
You need to add yourself to the [maintainers file.](https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix#L1)

Reply:
Btw this might be what's causing “This PR does not cleanly list package outputs after merging.”

Reply:
done, thanks!